Tests/TuistKitTests/Cache/CacheControllerTests.swift

```diff
@@ -61,16 +61,24 @@ final class CacheControllerTests: TuistUnitTestCase {
         let path = try temporaryPath()
         let xcworkspacePath = path.appending(component: "Project.xcworkspace")
         let project = Project.test(path: path, name: "Cache")
-        let aTarget = Target.test(name: "A")
-        let bTarget = Target.test(name: "B")
-        let aFrameworkPath = path.appending(component: "A.framework")
-        let bFrameworkPath = path.appending(component: "B.framework")
+        let targetNames = ["foo", "bar", "baz"].shuffled()
```

Comment:
I'd probably give this a static order rather than using `shuffled()` - the reason is that this might make the test flaky (although the chances of that are not very high)

Reply:
You can see that later I refer to it as the `aTarget.name`, so it should not make the test flaky, in fact, it tests that it is using correctly the topological sort, so that only the which targets depends on which, is important rather then how the targets were listed 😄 It should not fail because of the shuffled, otherwise there is something incorrect in either the test or the logic!

Reply:
ah, I did not realize that this is just used for naming, so it's fine to keep it as is it is then 🙂